bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BoschHttpClient.java

```diff
@@ -126,10 +126,9 @@ public boolean doPairing() {
             items.put("primaryRole", "ROLE_RESTRICTED_CLIENT");
             items.put("certificate", "-----BEGIN CERTIFICATE-----\r" + publicCert + "\r-----END CERTIFICATE-----");
 
-            String url = this.createSmartHomeUrl("clients");
-            Request request = this.createRequest(url, HttpMethod.POST, new StringContentProvider(gson.toJson(items)))
-                    .header("Systempassword",
-                            Base64.getEncoder().encodeToString(systempassword.getBytes(StandardCharsets.UTF_8)));
+            String url = this.createPairingUrl();
```

Comment:
Ah, this was probably the main reason why the pairing didn't work anymore. Sorry for introducing the bug and thanks for finding it :)